src/Compilers/CSharp/Test/Symbol/Compilation/GetSemanticInfoTests.cs

```diff
@@ -6003,5 +6003,29 @@ class B<T, U, U>
             symbol = model.GetDeclaredSymbol(typeParameters[typeParameters.Length - 2]);
             Assert.True(symbol.IsReferenceType);
         }
+
+        [Fact]
+        [WorkItem(25262, "https://github.com/dotnet/roslyn/issues/25262")]
+        public void IncompleteInvocationSpeculativeSymbolInfo()
+        {
+            var source = @"
+class Foo
+{
+    void Bar(string s)
+    {
+        s?.ToString();
+    }
+}";
+
+            var comp = CreateCompilation(source);
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree);
+
+            var invocation = tree.GetRoot().DescendantNodes().OfType<InvocationExpressionSyntax>().First();
+            var newInvocation = invocation.WithoutTrivia();
```

Comment:
Can you please remove this call? I don't believe it's necessary, and if it _is_ necessary I'd like to understand why.

Reply:
It is necessary to disconnect the node from its `Parent`. Without this, `FindConditionalAccessNodeForBinding` is able to walk up the tree to find the `ConditionalAccessExpressionSyntax`. I have added a comment clarifying that.

Reply:
> It is necessary to disconnect the node from its Parent. Without this, FindConditionalAccessNodeForBinding is able to walk up the tree to find the ConditionalAccessExpressionSyntax. I have added a comment clarifying that.

@svick this comment right here is why I don't necessarily feel comfortable with this change. The parser does not produce trees like this, and we do not generally harden the binder against syntax that is not produced by standard parsing.

Reply:
@333fred What is the alternative? I think Roslyn should never throw `NullReferenceException` from its public API. And while I would be fine if it threw a more appropriate exception here (probably `InvalidOperationEcxception`), doing that is probably the same or even slightly more effort (due to exception message localization) than just making it work.

Reply:
Frankly, there are lots of things that users can do in terms of invalid syntax trees, and it's a non-goal to try and harden ourselves to every invalid input. We make a ton of assumptions in the compiler around "this node will never have a `null` child because the parser won't do that", and this is no different. There's certainly an argument to be had that this particular form of invalid input is more common: it's been hit by both the IDE and a separate customer. However, I don't think I want to just bind it as a bad expression. That could very well cover up other bugs because that bad expression will return _something_, and whatever consumer is asking for this information will get that something and not necessarily know that it's bad.